### ISSUE TITLE
ISO 180004

### ISSUE THREAD

**Issue by sloopsight** [nayuki/Fast-QR-Code-generator#4]
Is the QR code generated by this lib is ISO 180004 complaint?

**Comment by nayuki** [comment]
It looks like you are talking about "ISO/IEC 18000-4:2018 | Information technology — Radio frequency identification for item management — Part 4: Parameters for air interface communications at 2.45 GHz".

I've never heard of it and it doesn't relate to QR Codes, so I'm pretty sure the answer is no.

**Comment by nayuki** [comment]
Or wait, I think you mean [ISO/IEC 18004:2006 Information technology — Automatic identification and data capture techniques — QR Code 2005 bar code symbology specification](https://www.iso.org/standard/43655.html)?

I read an unofficial copy of the ISO 18004 QR Code specification document. For the features I chose to support, I have implemented them faithfully. I omitted certain lesser-used features such as structured append and Micro QR Codes. So practically speaking, yes, the QR Codes generated by my library are compliant with ISO 18004.

However, I do not have an official certification or anything like that, so I am not allowed to advertise or claim that my work is ISO 18004 compliant. I believe most other open-source libraries are in the same boat too; they generate correct barcodes but are cannot claim to be *officially* ISO 18004 compliant.
